special_character_list

```diff
@@ -1,2197 +1,2187 @@
-alpha   ga     :α
```

Comment:
Why is this entire file different?

Reply:
Because the delimiter is tab now (see the commit message).

I also took the liberty to remove underscores at a few places, and putting alternatives between parentheses just for the look and feel.

Reply:
Gotchya, seems alright. Thanks!